selfdrive/controls/controlsd.py

```diff
@@ -471,6 +481,8 @@ def state_transition(self, CS):
         else:
           if self.events.any(ET.PRE_ENABLE):
             self.state = State.preEnabled
+          elif self.events.any(ET.OVERRIDE):
+            self.state = State.overriding
```

Comment:
add override alert types

Reply:
I think here we want to show enable first? preEnable doesn't add alert type here